pkg/schemes/scheme.go

```diff
@@ -64,11 +64,15 @@ func DefaultScheme() *runtime.Scheme {
 	return s
 }
 
-// TestingScheme unconditionally includes the default scheme and the gwv1a2 scheme (which includes TCPRoute).
-func TestingScheme() *runtime.Scheme {
```

Comment:
Given that this is exported should we leave it in and mark it deprecated? otherwise anything that relies on this will break

Reply:
I don't think so since the function was recently added and never included in a previous major or minor release.